README.md

```diff
@@ -9,13 +9,25 @@ Pre-boot authentication image for TCG Storage devices
 ```shell
 $ sudo apt install \
     gnupg2 gpgv2 flex bison build-essential libelf-dev \
-    curl libssl-dev bc zstd dosfstools gdisk mtools
+    curl libssl-dev bc zstd dosfstools fdisk gdisk mtools
 $ gpg2 --locate-keys torvalds@kernel.org gregkh@kernel.org autosigner@kernel.org
 
 # Make sure sgdisk is in the PATH
 $ PATH=$PATH:/sbin make
 ```
 
+Alternatively, use the containerized build tools:
+
+```shell
+$ docker build \
+	-t elx-pba-builder:latest \
```

Comment:
WDYT about making these fully qualified e.g. `elastx.se/elx-pba-builder`? It would avoid somebody snatching elx-pba-builder on dockerhub, I think.